### ISSUE TITLE
Plugin config example missing recipients field in config.go

### ISSUE THREAD

**Issue by inertial-frame** [gravitational/teleport-plugins#251]
The Slack Plugin is working great. However, the following config example lacks a `recipients` field:

https://github.com/gravitational/teleport-plugins/blob/a89a239e053414b4a085dd66a80b9f93407276d8/access/slack/config.go#L24-L37

WIthout this field added to the config I get the following after running:

```bash
sudo tctl request create test-user --roles=admin 3daf04b3-7780-4cff-a1ce-345c0318776f
```

```bash
No channel to post request_id:34cac0ad-580e-4b37-9a5c-85e7d90da7e7 request_op:put slack/app.go:209
```

Furthermore, the example should more helpfully indicate:

1. That the recipient format for users should be the email address.
2. That the recipient format for a channel should be the channel ID.

```
recipients=["person@place.com","AIBOEFJD"]
```

**Comment by inertial-frame** [comment]
Verified this for 6.2.3:

```
WARN   No channel to post request_id:ab1903b8-98db-456d-89d6-71fa3418f1ae request_op:put slack/app.go:209
```

The TOML is also updated here: https://github.com/gravitational/teleport/pull/7041

**Comment by inertial-frame** [comment]
Updated this to include `suggested_reviewers` as well.

**Comment by berdee** [comment]
fixed in #259
